exchanges/assets.py

```diff
@@ -0,0 +1,49 @@
+class Asset(object):
+
+    def __init__(self, name, atype, iso4217_a3=None, abbreviation=None):
+
+        if iso4217_a3 is None and abbreviation is None:
+            raise ValueError('At least one denomination is needed')
+
+        if atype not in ['X', 'Z']:
+            raise ValueError("atype must be either 'X' or 'Z'")
```

Comment:
Do X and Z have semantic meaning? If so, we could add an enumeration (new in python3) for them or define class constants.